tool/tsh/common/proxy_test.go

```diff
@@ -1020,15 +1024,17 @@ func mustLogin(t *testing.T, s *suite, args ...string) (tshHome, kubeConfig stri
 
 // login with new temp tshHome and set it in Env. This is useful
 // when running "ssh" commands with a tsh "ProxyCommand".
-func mustLoginSetEnv(t *testing.T, s *suite, args ...string) (tshHome, kubeConfig string) {
-	tshHome, kubeConfig = mustLogin(t, s, args...)
+// deprecated: Create a new helper that depends on mustLogin instead which requires migrating from
```

Comment:
```suggestion
// TODO(ravicious): Create a new helper that depends on mustLogin instead which requires migrating from
```

Did you intend to follow up on this TODO in this PR or just leave a note?

Reply:
Just leave a note. `testenv.MakeTestServer` has been there for quite some time, so I just wanted to make sure that the next person who's going to copy paste some test structure for another tsh test is aware that functions that depend on `newTestSuite` are deprecated too.